app/controllers/admin/communities_controller.rb

```diff
@@ -9,11 +9,7 @@ def getting_started
     @selected_left_navi_link = "getting_started"
     @community = @current_community
 
-    if(feature_enabled?(:onboarding_redesign_v1))
-      redirect_to admin_getting_started_guide_path
-    else
-      render locals: {paypal_enabled: PaypalHelper.paypal_active?(@current_community.id)}
-    end
+    redirect_to admin_getting_started_guide_path
```

Comment:
That can be also done in a separate PR if we just want to quickly enable onboarding for everyone, but anyhow it's good to do it.